libcaliptra/src/caliptra_mbox.h

```diff
@@ -128,7 +128,7 @@ static inline bool caliptra_mbox_is_busy(void)
 
 static inline uint8_t caliptra_mbox_read_status_fsm(void)
 {
-    return (uint8_t)(caliptra_mbox_read(MBOX_CSR_MBOX_STATUS) >> 16 & MBOX_CSR_MBOX_STATUS_STATUS_MASK);
+    return (uint8_t)(caliptra_mbox_read(MBOX_CSR_MBOX_STATUS) >> 6) & 7;
```

Comment:
what are 6 and 7 ?

Reply:
i changed it to use the #defines available in the headers.